### ISSUE TITLE
Overlap of standard cells in GDS of TinyRocket generated from the ASAP7 VLSI Flow

### ISSUE THREAD

**Issue by baichen318** [ucb-bar/chipyard#1111]
Hi all,

Recently I encountered a problem with the generated GDS of TinyRocket.

I found that layer `7/0​` in GDS overlapped.

A screenshot below illustrates an abnormal case in my generated GDS of TinyRocket.
<img width="1920" alt="abnormal-overlap" src="https://user-images.githubusercontent.com/33573154/153330787-96eb4c97-7a05-46dd-ace5-d1f6b3f5958f.png">

For each standard cell, each component of layer `7/0`​ has approximately the same gap (AND2x2_ASAP7_75t_L), as a screenshot shown below.
<img width="1920" alt="AND2x2_ASAP7_75t_L" src="https://user-images.githubusercontent.com/33573154/153330839-e8923a9b-eb8a-4b46-9823-986a2788f440.png">

However, multiple standard cells overlap each other in the final GDS.

The VLSI flow to obtain the GDS of TinyRocket strictly follows the document: https://chipyard.readthedocs.io/en/latest/VLSI/Tutorial.html.

I wondered what caused the abnormal case and how I could solve it.

Thank you very much in advance!

Best regards,
Chen

**Comment by harrisonliew** [comment]
Hi @baichen318, I have received your email about this but didn't have time to look at it yet. I will try to debug tomorrow and let you know.

**Comment by harrisonliew** [comment]
Hi @baichen318,

I have figured out that it actually is due to the ASAP7 PDK r1p5. I was able to reproduce your GDS using the current Chipyard release. The issue is caused by some standard cell LEFs having incorrect boundaries.

On the `dev` branch, the tutorial is updated for the publicly released ASAP7 PDK r1p7 (#934). Here is what my gate layer shows in this new PDK:

![image](https://user-images.githubusercontent.com/11860787/153652656-37698f0b-7419-43b8-a671-f45ab36c0ba2.png)

We plan to release Chipyard pointing to this newer PDK very soon.

-Harrison
